HOWTO/LICENSE-HEADERS.md

```diff
@@ -61,7 +61,9 @@ limitations under the License.
 ```
 
 If the file is under some other license, it needs to have its `SPDX-License-Identifier`
-and also a copy of the license information needs to be in `scripts/license-header-templates/`.
+and also a copy of the license header information needs to be in `LICENSES/HEADERS/`.
+If the entire license is to be part of the header, you should place the license
+text in `LICENSES/` directly and the tool will use those.
```

Comment:
This is a good explanation. It leaves open for people to know when the entire license needs to be part of the header, and when not. I am not sure if there are guidelines, but it would be good provide some link or information as to when one case applies. Otherwise, we are leaving this up-to the user.

Reply:
I don't think there is a general consensus except that when the license is "small enough" to fit in a header it should be in a header, other wise there should be a text with a link. I've added that clarification.